docs/HowTo/topics/Topic-overview.md

```diff
@@ -0,0 +1,50 @@
+# Topic Overview
+
+Topic overview provides different views of topic. 
+
+## Overview
+
+In this tab, the below are displayed
+
+- topic configuration like partitions, replication factor and if any advanced configuration
+- cluster details like bootstrap url
+- count of producer and consumer subscriptions and schemas
+
+
+## Subscriptions
+
+In this tab, all producer and consumer acls are displayed, including prefixed and transactional.
+
+
+## Messages
+
+Plain events can be viewed if they exist on the topic. Note that a consumer group has to be configured in cluster api, 
+to view events.
+
+    klaw.topiccontents.consumergroup.id
+
+
+## Schemas
+
+Avro schemas and all available versions of the subject (topic) can be viewed in Schemas tab. It is also possible to 
+request for a new schema and promote the schema.
+
+
+## Readme
+
+Any information about the topic, can be documented here, and will be searchable.
```

Comment:
A feature rich Markdown documentation tool for any Information about the topic that sohuld be documented and it will be fully searchable.